app/create_file.py

```
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
+        index = 0
+
+        for _ in range(len(argv) - 1):
+            index += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index:len(argv)]
+
+file_data = [datetime.today().strftime('%y:%d:%m %H-%M-%S'), "\n"]
+
+while True:
+    line = input("Enter content line: ")
+    if line == "exit":
+        break
+    file_data.extend([line])
+
+for directory in directories:
+    try:
+        mkdir(directory)
+    except Exception:
```

Code is using a classic generic try-except : `except Exception: pass` for directory creation. Not good idea, that is silent swallowing all errors and making debugging harder. Rather, handle specific exceptions and return proper error messages. 
```

try:
    if not os.path.exists(directory):
        os.mkdir(directory)
        print(f"Directory '{directory}' created.")
    else:
        print(f"Directory '{directory}' already exists.")
except OSError as e:
    print(f"Error creating directory '{directory}': {e}")
    exit(1)
```